### ISSUE TITLE
Why Fork?

### ISSUE THREAD

**Issue by mahenzon** [mahenzon/flask-bcrypt#2]
**TLDR**: to keep the repo [updated at PyPI](https://pypi.org/project/Bcrypt-Flask/).

Original source was not updated on [PyPI](https://pypi.org/project/Flask-Bcrypt/) since **Oct 6, 2015**. But the [source](https://github.com/maxcountryman/flask-bcrypt) gets some updates from time to time. I suggested updating the repo on PyPI and [provided prepared actions](https://github.com/maxcountryman/flask-bcrypt/pull/66), but author refused to update source at PyPI and suggested installing the lib from GitHub, but even the version doesn't get bumped up at all. He also asked me if I wanna pay him for updating the lib at PyPI, but closed conversation, so I can't ask for the rate.

Anyway, I wanna keep my requirements file simple and clean, so I published an updated version of Flask-Bcrypt [at PyPI as `Bcrypt-Flask`](https://pypi.org/project/Bcrypt-Flask/) (import name stays the same, this is the only change for PyPI).

This closes my issue https://github.com/maxcountryman/flask-bcrypt/issues/65 and an earlier one https://github.com/maxcountryman/flask-bcrypt/issues/53 (mine was a duplicate).

**Comment by seanieb** [comment]
Thanks folks, this was badly needed and the previous maintainer had a super bad attitude.

**Comment by adonai001** [comment]
When we tried to move to this, deployment is failing with "AttributeError: 'LocalStack' object has no attribute '__ident_func__' any suggestions?

**Comment by mahenzon** [comment]
@adonai001 hey! could you please share your dependencies (Flask, Werkzeug) and full traceback? maybe we need to update the lib to support Werkzeug >= 2.0

**Comment by alanhamlett** [comment]
Version 1.0.0 of [Flask-Bcrypt](https://pypi.org/project/Flask-Bcrypt/) was released today. Sorry it took so long!

**Comment by mahenzon** [comment]
Wow, great! Finally someone could handle it!
